src/services/db/RepoService.ts

```diff
@@ -144,6 +151,51 @@ export default class RepoService extends GitHubService {
     })
   }
 
+  async readMediaFile(
+    sessionData: UserWithSiteSessionData,
+    { fileName, directoryName }: { fileName: string; directoryName: string }
+  ): Promise<any> {
```

Comment:
nit: Are we able to specify the return type other than `any`?

Reply:
I think we can specify the type like:
`Promise<MediaFileOutput, GitFileSystemError | NotFoundError>`